packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -336,6 +336,18 @@ export interface CommonAutoScalingGroupProps {
    * @default - `TerminationPolicy.DEFAULT`
    */
   readonly terminationPolicies?: TerminationPolicy[];
+
+  /**
+   * The amount of time, in seconds, until a newly launched instance can contribute to the Amazon CloudWatch metrics.
+   * This delay lets an instance finish initializing before Amazon EC2 Auto Scaling aggregates instance metrics,
+   * resulting in more reliable usage data. Set this value equal to the amount of time that it takes for resource
+   * consumption to become stable after an instance reaches the InService state.
+   *
+   * @see https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-default-instance-warmup.html
+   *
+   * @default None
```

Comment:
is this what we decided on? if so, I don't like calling the default none. That's not very descriptive for the user, because what does none imply? there's no default instance warmup? or that the value is 0? we can afford to be a bit more descriptive here (and i know we say `None` all over the place, but we should hold ourselves to that standard :) ).